Dockerfile

```diff
@@ -1,12 +1,12 @@
-# Use following version of Node as the base image
-FROM node:10
+# Use following version of Node alpine as the base image
+FROM node:10-alpine
 
 # Set work directory for run/cmd
 WORKDIR /app
 
 # Copy package.json into work directory and install dependencies
 COPY package.json /app/package.json
```

Comment:
can you copy package-lock.json as well here? then we are sure that the exact same package versions are installed within the docker container as were installed during the tests.